packages/webapp-libs/webapp-tenants/src/components/tenantMembersList/membershipEntry/membershipEntry.component.tsx

```diff
@@ -0,0 +1,92 @@
+import { TenantMembershipType, TenantUserRole } from '@sb/webapp-api-client';
+import { Button } from '@sb/webapp-core/components/buttons';
+import {
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuLabel,
+  DropdownMenuPortal,
+  DropdownMenuSub,
+  DropdownMenuSubContent,
+  DropdownMenuSubTrigger,
+  DropdownMenuTrigger,
+} from '@sb/webapp-core/components/dropdownMenu';
+import { TableCell, TableRow } from '@sb/webapp-core/components/table';
+import { GripHorizontal, Hourglass, UserCheck } from 'lucide-react';
+import { indexBy, prop, trim } from 'ramda';
+import { useMemo } from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import { useTenantRoles } from '../../../hooks/useTenantRoles';
+
+export type MembershipEntryProps = {
+  className?: string;
+  membership: TenantMembershipType;
+};
+
+export const MembershipEntry = ({ membership, className }: MembershipEntryProps) => {
+  const { getRoleTranslation } = useTenantRoles();
+
+  const roles = [TenantUserRole.OWNER, TenantUserRole.ADMIN, TenantUserRole.MEMBER];
+  const roleChangeCallbacks = useMemo(() => {
+    const mapper = roles.map((role) => ({
+      role,
+      callback: () => console.log({ role }),
+    }));
+    return indexBy(prop('role'), mapper);
+  }, [roles, membership.id]);
+
+  const name = trim([membership.firstName, membership.lastName].map((s) => trim(s ?? '')).join(' '));
+  return (
+    <TableRow className={className}>
+      <TableCell>{name || membership.userEmail || membership.inviteeEmailAddress}</TableCell>
+      <TableCell>{getRoleTranslation(membership.role?.toUpperCase() as TenantUserRole)}</TableCell>
```

Comment:
This is generated based on the GraphQL schema. Unfortunately backend is returning there a string instead of enum:( I was talking already about this with @wojcikmat and he already spent some time to fix this but without any luck:/ This was the best approach for now.